Lombiq.Tests.UI/Extensions/ScreenshotUITestContextExtensions.cs

```diff
@@ -24,11 +31,25 @@ public static class ScreenshotUITestContextExtensions
     /// </summary>
     public static Bitmap TakeElementScreenshot(this UITestContext context, IWebElement element)
     {
-        using var screenImage = context
-            .TakeScreenshot()
-            .ToBitmap();
+        var originalScrollPosition = context.GetScrollPosition();
+
+        try
+        {
+            var decorator = new VerticalCombineDecorator(new ScreenshotMaker())
+                .SetWaitAfterScrolling(TimeSpan.FromMilliseconds(ScrollDelay));
```

Comment:
Is this a fixed 1 s wait? Because we shouldn't rely on such things, which might or might not work. Instead of waiting for a given amount of time regardless of the context, we should wait for the desired action to be completed, i.e. verifying somehow that it completed, with a timeout.

Reply:
If we implement this feature, I'll use `WaitScrollToNotChange` instead of fixed time.